docs/docs/en/guide/project/workflow-definition.md

```diff
@@ -74,14 +74,22 @@ The following are the operation functions of the workflow definition list:
     * Recipient: select notification policy||timeout alarm||when fault tolerance occurs, process result information or alarm email will be sent to the recipient list
     * Cc: select notification policy||timeout alarm||when fault tolerance occurs, the process result information or warning email will be copied to the CC list
     * Startup parameter: Set or overwrite global parameter values when starting a new process instance
-    * Complement: two modes including serial complement and parallel complement. Serial complement: within the specified time range, the complements are executed from the start date to the end date and N process instances are generated in turn; parallel complement: within the specified time range, multiple days are complemented at the same time to generate N process instances.
+    * Complement: two modes including serial complement and parallel complement. Serial complement: within the specified time range, the complements are executed from the start date to the end date and N process instances are generated in turn; parallel complement: within the specified time range, multiple days are complemented at the same time to generate N process instances.Manual input, serial complement: manually enter the date in comma separated date format as `yyyy-MM-dd HH:mm:ss`, and generate n process instances in turn; Parallel Replenishment: manually enter a date in the comma separated date format of `yyyy-MM-dd HH:mm:ss`. Replenishment is performed for multiple days and n process instances are generated at the same time.
```

Comment:
add blanket before 'Manual input,'

Reply:
please use consist word 'complement' instead of 'Replenishment'
and avoid use passive pattern

Reply:
ok,modified